google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugGlobalPoller.java

```diff
@@ -166,13 +170,27 @@ private void fireBreakpointsChanged(@NotNull CloudDebugProcessState state) {
    *
    * @param state represents the target debuggee to query
    */
-  private void pollForChanges(@NotNull final CloudDebugProcessState state) {
+  void pollForChanges(@NotNull final CloudDebugProcessState state) {
     final Debugger client = CloudDebuggerClient.getShortTimeoutClient(state);
     if (client == null) {
-      // It is ok if there is no client.  We may want to consider notifying the user that
-      // background polling is disabled until they login, but for now, we can just doc it.
-      LOG.info("no client available attempting to checkForChanges");
-      return;
+      if (state.isListenInBackground()) {
```

Comment:
On line 174 we failed to obtain a Debugger client, presumably because of access error (or e.g. network failure). If state.isListenInBackground() is true it means that this debug session is listening for events in the background. Thus if it is supposed to listen but cannot get access view the Debugger client, that's an unexpected situation.